### PR TITLE
Snhut 3114 -  Evaluated assessments

### PR DESCRIPTION
**Pull request by shafi-sadaindia**
### Description

- As an assessor, I need the ability to find assessments I have evaluated, So that I can review prior submissions. 
- As an assessor, I need the ability to find assessments all evaluated assessments, So that I can review prior submissions. 

#### Screenshots

Screenshot 2023-01-11 at 1 55 21 PM
Screenshot 2023-01-11 at 1 55 14 PM

### Other Details

- [JIRA Ticket](https://collectiveshift.atlassian.net/browse/SNHUT-3114)

#### Checklist

- [x] **Assumptions of User Story met (including acceptance criteria)**
- [x] **All new necessary Unit tests were CREATED** to promote code coverage
- [x] **All required unit tests are PASSING**
- [x] The proposed functionality was tested manually locally, and it's working fine.
- [ ] PR was properly reviewed by a team member (at least 1 approval required to merge!)
- [x] Unnecessary console.logs AND/OR comments were removed
- [ ] Feature ok-ed by UX designer (If needed - not for back end components) and Product Owner
- [ ] Ensure configuration and/or build changes documented

### CODE REVIEW

File: apps/toboggan-app/src/app/assessment/components/all-evaluations-list/all-evaluations-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+  ngOnDestroy(): void {
+    [this.datageneratorSubscription, this.updateEvaluatedAssessmentSubscription].map((s) => {
+      if (s.unsubscribe) s.unsubscribe();
+    });
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  getActionMenuItems = () => {
+    return ['view details', 'edit', 'delete'];
```
Comment (sada-ajith) [inline apps/toboggan-app/src/app/assessment/components/all-evaluations-list/all-evaluations-list.component.ts:50]: Can we verify that these actions are present in table. ?

File: apps/toboggan-app/src/app/assessment/components/my-evaluations-list/my-evaluations-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+  ngOnDestroy(): void {
+    [this.datageneratorSubscription, this.updateEvaluatedAssessmentSubscription].map((s) => {
+      if (s.unsubscribe) s.unsubscribe();
+    });
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  getActionMenuItems = () => {
+    return ['view details', 'edit', 'delete'];
```
Comment (sada-ajith) [inline apps/toboggan-app/src/app/assessment/components/my-evaluations-list/my-evaluations-list.component.ts:50]: Can we verify that these actions are present in table. ?

**Review by sada-ajith — CHANGES_REQUESTED** [review]
Could you please note and verify item I have mentioned. ?

File: apps/toboggan-app/src/app/assessment/components/all-evaluations-list/all-evaluations-list.component.ts (hunk 1 above)
Comment (shafi-sadaindia) [reply apps/toboggan-app/src/app/assessment/components/all-evaluations-list/all-evaluations-list.component.ts:50]: View details only. edit and delete is not covered in user story. However we can keep it.

File: apps/toboggan-app/src/app/assessment/components/my-evaluations-list/my-evaluations-list.component.ts (hunk 2 above)
Comment (shafi-sadaindia) [reply apps/toboggan-app/src/app/assessment/components/my-evaluations-list/my-evaluations-list.component.ts:50]: View details only. edit and delete is not covered in user story. However we can keep it.

**Review by sada-ajith — APPROVED** [review]

**Pull request merged**